avocado/core/test.py

```diff
@@ -139,7 +139,7 @@ def record_and_warn(*args, **kwargs):
 
         self.debugdir = None
         self.resultsdir = None
-        self.status = None
+        self.status = "ERROR"
```

Comment:
Why do you think so? It's common everywhere, the test errors, unless it reaches the point to make it pass. It's the same as the default exit status, it's fail, unless it passes.

Also `None` is not a valid status, so if we did it properly and used `enum` to assign statuses, we'd have to pick one and "ERROR" is the closest valid status I know.